code/modules/urist/modules/newtrading/trade/trade_items/artifact dealer.dm

```diff
@@ -0,0 +1,154 @@
+/datum/trade_item/artifacts
+	quantity = 0
+	trader_weight = 1
+	value = 500
+
+/datum/trade_item/artifacts/tape
+	name = "retro holotape"
+	category = "artifacts"
```

Comment:
If all items share this category (which it appears they do), why not set it in `datum/trade_item/artifacts` instead?

Reply:
Im not sure i understand what you mean

Reply:
oh do you mean the line `category = `?
because i copied it from glloyd

Reply:
under `datum/trade_item/artifacts` define the category there, rather than for every subtype
ie
```
/datum/trade_item/artifacts
	//[...]
	category = "artifacts"

/datum/trade_item/artifacts/tape
	name = "retro holotape"
	//removed category from here
```
So you don't have to define it each time for each and every item. Just once for the parent type